code/modules/projectiles/guns/energy/experimental.dm

```diff
@@ -15,7 +15,36 @@
 	icon = 'icons/obj/gun_experimental.dmi'
 	icon_state = "bison"
 	item_state = null
-	projectile_type = "/obj/item/projectile/bison"
+	projectile_type = "/obj/item/projectile/beam/bison"
 	charge_cost = 100
-	cell_type = "/obj/item/weapon/cell/bison"
-	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/experimentals.dmi', "right_hand" = 'icons/mob/in-hand/right/experimentals.dmi')
+	cell_type = "/obj/item/weapon/cell"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/guns_experimental.dmi', "right_hand" = 'icons/mob/in-hand/right/guns_experimental.dmi')
+	fire_delay = 8
+	var/pumping = 0
+
+/obj/item/weapon/gun/energy/bison/New()
+	..()
+	power_supply.charge = 0
+
+/obj/item/weapon/gun/energy/bison/attack_self(mob/user as mob)
```

Comment:
I know, hence why I made a pumping var here to prevent people from spamming it too fast.